server/src/http/controllers/formations.controller.v2.ts

```diff
@@ -92,21 +53,26 @@ export default (server: Server) => {
     async (req, res) => {
       const { id } = req.params
       const { caller } = req.query
+      const api_path = "/v2/formations/formation/:id"
       try {
-        const result = await getFormationQuery({ id })
+        const formationOpt = await getFormationv2({ id })
         if (caller) {
           trackApiCall({
             caller,
-            api_path: "formationV1/formation",
+            api_path,
             training_count: 1,
             result_count: 1,
             response: "OK",
           })
         }
-        return res.send(result)
+        if (formationOpt) {
+          return res.send(formationOpt)
```

Comment:
déplacer le trackApiCall ici pour éviter de tracker les not found comme des success